foxpuppet/browser/navbar.py

```diff
@@ -4,33 +4,30 @@
 
 
 from selenium.webdriver.common.by import By
-from selenium.webdriver.support.ui import WebDriverWait
-from selenium.webdriver.support import expected_conditions as EC
 
 
 class Navbar(object):
+
+    _file_menu_button_locator = (By.ID, 'file-menu')
+    _file_menu_private_window_locator = (By.ID, 'menu_newPrivateWindow')
+    _file_menu_new_window_button_locator = (By.ID, 'menu_newNavigator')
+    _bookmark_menu_button_locator = (By.ID, 'bookmarks-menu-button')
+
     def __init__(self, selenium, *args, **kwargs):
         self.selenium = selenium
         self._locationbar = None
-        self._menu_button = (By.ID, 'PanelUI-menu-button')
 
     def open_window(self, private=False):
```

Comment:
Since when is this method part of the NavBar? Did I miss a move? From my perspective it is still a method which should be attached to the browser window.